actioncable/lib/action_cable/channel/broadcasting.rb

```diff
@@ -7,22 +7,26 @@ module Channel
     module Broadcasting
       extend ActiveSupport::Concern
 
-      delegate :broadcasting_for, to: :class
+      delegate :broadcasting_for, :broadcast_to, to: :class
 
       module ClassMethods
         # Broadcast a hash to a unique broadcasting for this <tt>model</tt> in this channel.
         def broadcast_to(model, message)
-          ActionCable.server.broadcast(broadcasting_for([ channel_name, model ]), message)
+          ActionCable.server.broadcast(broadcasting_for(model), message)
         end
 
         def broadcasting_for(model) #:nodoc:
```

Comment:
If this method is intended to be used in app tests, it should be documented.

Reply:
Done.
Added a note to the change log.